.azure-devops/templates/run-tests-parallel.yml

```diff
@@ -79,23 +82,28 @@ steps:
         azureSubscription: ${{ parameters.azureSubscription }}
         scriptType: bash
         scriptLocation: inlineScript
+        arguements: azCliArguments
```

Comment:
Misspelling on arguments here, but more importantly why is this being introduced?

Reply:
Was messing with yaml - undid changes